src/webview/common/vscode.css

```diff
@@ -86,6 +86,7 @@ textarea::placeholder {
 .contain-style {
 	top: 2%;
     position: relative;
+	padding-left: 5%;
```

Comment:
Maybe better to use a fixed value instead of percentage otherwise you have an ugly behavior when resizing the tab. All the other vscode elements have a fixed left margin/padding 
![moving-message](https://user-images.githubusercontent.com/49404737/204496342-7b8897a0-eb0e-4a8e-89da-84d2f5a2d6f8.gif)